app/models/watched_movies.rb

```diff
@@ -0,0 +1,7 @@
+class Watched Movie < ApplicationRecord
```

Comment:
半角スペースが入っていますよ！

```suggestion
class WatchedMovie < ApplicationRecord
```

さらに，ファイル名が間違っています。モデルは「単数形」です。

```
watched_movies.rb --> watched_movie.rb
```